internal/resource/nvml-mig-device.go

```diff
@@ -132,3 +133,23 @@ func totalMemory(attr map[string]interface{}) (uint64, error) {
 		return 0, fmt.Errorf("unsupported attribute type %v", t)
 	}
 }
+
+func (d nvmlMigDevice) GetPIEClass() (uint32, error) {
+	info, retVal := d.MigDevice.GetPciInfo()
```

Comment:
Question: Is PCI info valid for a MIG device? How does the busid differ from that of the parent?